lib/zendesk_api/client.rb

```diff
@@ -142,6 +143,7 @@ def build_connection
         builder.use ZendeskAPI::Middleware::Response::Logger, config.logger if config.logger
         builder.use ZendeskAPI::Middleware::Response::ParseIsoDates
         builder.use ZendeskAPI::Middleware::Response::ParseJson
+        builder.use ZendeskAPI::Middleware::Response::SanitizeResponse
```

Comment:
It is the first thing that runs. I've tested this locally and it seems that it runs in the opposite order we declare the middleware.